global.json

```diff
@@ -5,12 +5,7 @@
     "rollForward": "major"
   },
   "tools": {
-    "dotnet": "8.0.100-preview.7.23376.3",
-    "runtimes": {
-      "dotnet": [
-        "7.0.0"
-      ]
-    }
+    "dotnet": "8.0.100-preview.7.23376.3"
```

Comment:
Any reason not to use the RTM 8.0.100 SDK now?

Reply:
Not really - let me update this.